src/redis-check-aof.c

```diff
@@ -242,7 +242,7 @@ int checkSingleAof(char *aof_filename, char *aof_filepath, int last_file, int fi
     }
 
     if (preamble) {
-        char *argv[2] = {NULL, aof_filename};
+        char *argv[2] = {NULL, aof_filepath};
```

Comment:
please mention this change in the top comment (to be used as a squash-merge commit comment) so it'll be clear for future readers why we did that.

Reply:
ohh, i see you already did. thanks